metricbeat/module/elasticsearch/index_summary/data.go

```diff
@@ -47,6 +47,46 @@ var (
 					"bytes": c.Int("memory_in_bytes"),
 				},
 			}),
+			"indexing": c.Dict("indexing", s.Schema{
+				"index": s.Object{
+					"count": c.Int("index_total"),
+					"time": s.Object{
+						"ms": c.Int("index_time_in_millis"),
+					},
+				},
+				// following field is not included in the Stack Monitoring UI mapping
```

Comment:
You can find comments like this one around the fields that I couldn't see in the mappings that @chrisronline said that were required. If you see this comment it means that the field was present in `data_xpack.go` file but not in `data.go` file and it's not found in its respective `type` in the ES mapping

Reply:
Do you think we can remove these comments now, or right before merging this PR?

Reply:
I removed the comment